packages/signals/source/SignalRemoteReceiver.ts

```diff
@@ -158,26 +159,27 @@ export class SignalRemoteReceiver {
 
         return implementationMethod(...args);
       },
-      insertChild: (id, child, index) => {
+      insertChild: (id, child, parentRemoteId) => {
```

Comment:
`nextSiblingId` / `prevSilbingId`

Also check param name in other `insertChild` methods.